test/ProfilerTest.cpp

```diff
@@ -1,8 +1,105 @@
 #include "gtest/gtest.h"
 
 #include "pipert/Profiler.h"
-
+#include "pipert/Scheduler.h"
+#include "pipert/Profiler.h"
+#include <cstdint>
+#include <arpa/inet.h>
+#include <sys/types.h>
+#include <unistd.h>
 #include <cstdio>
+#include <thread>
+
+const std::uint16_t server_port = 8888;
```

Comment:
These should all go into anonymous namespace or mysterious things will happen again.